### PR TITLE
feat(APIM-261): API import post deployment

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction
Post deployment, APIM API service URL and latest endpoints are not imported as part of CICD.

## Resolution
* APIM API import command updates API service URL (latest revision FQDN).
* Same command also imports the API operations using OpenAPI 3 specification `/openapi/json` from the latest CA revision FQDN.
* Conditional local VNET peering to `pre-prod` or `prod` VNET.

## Miscellaneous
* Various outdated dependencies have been updated.

### CODE REVIEW

File: src/modules/healthcheck/healthcheck.controller.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export class HealthcheckController {
     private readonly mem: MemoryHealthIndicator,
   ) {}
 
-  @Get('ready')
+  @Get('API micro-service health check performs dependent databases connection and memory heap check')
```
Comment (IgnatG) [inline src/modules/healthcheck/healthcheck.controller.ts:25]: The `Get` decorator is used to create a path i.e. /api/v1/ready
This from above looks like the description of it

Comment (abhi-markan) [reply src/modules/healthcheck/healthcheck.controller.ts:25]: Ah apologies I'll double check I was planning to update the description

Comment (abhi-markan) [reply src/modules/healthcheck/healthcheck.controller.ts]: Have rectified.

**Review by IgnatG — APPROVED** [review]
LGTM

**Pull request merged**